modules/helm/template.go

```diff
@@ -33,6 +34,11 @@ func RenderTemplateE(t testing.TestingT, options *Options, chartDir string, rele
 		return "", errors.WithStackTrace(ChartNotFoundError{chartDir})
 	}
 
+	// check chart dependencies
+	if _, err := RunHelmCommandAndGetOutputE(t, &Options{}, "dependency", "build", chartDir); err != nil {
```

Comment:
Will be helpful to have a test case to keep track executions of `dependency build` for helm charts